src/Microsoft.Health.Fhir.Shared.Api/Microsoft.Health.Fhir.Shared.Api.projitems

```diff
@@ -9,10 +9,17 @@
     <Import_RootNamespace>Microsoft.Health.Fhir.Shared.Api</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)Controllers\AadSmartOnFhirProxyController.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Controllers\ExportController.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Controllers\FhirController.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Features\ActionResults\FhirResult.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Features\ActionResults\OperationOutcomeResult.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Features\Audit\AuditHelper.cs" />
```

Comment:
Why do these need to be moved to shared?

Reply:
I thought I left comment about the change but I guess I didn't.

The shared project (Microsoft.Health.Fhir.Shared.Api) has associated unit test projects (Microsoft.Health.Shared.Api.UnitTests) but the regular project doesn't. 

Some of the class in the Microsoft.Health.Fhir.Api had its unit tests in Microsoft.Health.Shared.Api.UnitTests which didn't seem right. So I moved those classes to the shared project. Alternatively, I would make another unit test project for Microsoft.health.Fhir.Api.

What's the guidance on what should go into shared and what shouldn't?

Reply:
With the new changes, this is no longer needed so I reverted back.